src/docs/asciidoc/work-with-jet/job-management.adoc

```diff
@@ -0,0 +1,37 @@
+[[command-line]]
+= Jet Command Line Tool
+
+Jet comes with a command line tool, `jet.sh` which can be used to submit jobs
+and do basic management of their lifecycle. The following commands are provided:
+
+[%header]
+|===
+|Command|Description
```

Comment:
I'd lose the alphabetical ordering and organize/group these commands by similarity.